cpp-client/deephaven/client/include/private/deephaven/client/impl/datetime_expression_impl.h

```diff
@@ -2,16 +2,13 @@
 
 #include <memory>
 #include <vector>
-#include "deephaven/client/highlevel/columns.h"
```

Comment:
I just noticed that you got rid of "highlevel" etc; care to elaborate a bit?

Reply:
In the previous version of the C++ library (the one that spoke the GWT protocol to Deephaven Enterprise) there was a pretty clear distinction between "low level" and "high level" interactions. In low-level interactions you could issue server commands yourself (various GWT calls... where I interpret GWT as the precursor to GRPC). In high-level interactions you would get your data wrapped up in ColumnSources, have Table abstractions, have automatic lazy evaluation, and also you would get all stuff in the "fluent" layer (the operator overloading stuff). The highlevel stuff was a layer on top of the lowlevel stuff.

For this version I decided that we weren't really getting any value out of the distinction between highlevel vs lowlevel. For example, we're not offering our users the ability to send their own GRPC messages to the server via our library. So for these reasons I ripped it out.